lib/server/index.js

```diff
@@ -60,9 +62,15 @@ app.use('/graphql', graphqlHTTP(req => ({
 })));
 
 app.use(async (req, res, next) => {
+
+  const metaSettings = {
+    'description': 'description', 'keywords': 'keywords',
+    'copyright':'copyright', 'robots':'robots', 'dctitle': 'DC.title'
+  };
+
   const settingsArr = await SettingModel
     .find({
-      _id: {$in: ['title', 'favicon']}
+      _id: {$in: ['title', 'favicon'].concat(_keys(metaSettings))}
```

Comment:
Use Object.keys instead. You can drop the lodash.keys dep. You can also use es6 spread:

```js
{$in: ['title', 'favicon', ...Object.keys(metaSettings)]}
```